### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by wartburggraaf** [nl-design-system/backlog#343] [Component suggestions]
<img width="480" alt="verschillende logo's bij elkaar van onder andere Amsterdam, Den Haag en Utrecht" src="https://github.com/user-attachments/assets/70a8db92-78b5-464c-919e-961c306ab18d">

We willen graag meer leren over de Logo component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1727875326722969)

---

## Naam
Logo

## Doel
Afbeelding met het beeldmerk of woordmerk van een organisatie.

## Onderzoek

[Website Logo Placement for Maximum Brand Recall - NN Group](https://www.nngroup.com/articles/logo-placement-brand-recall/)
In dit artikel wordt beschreven dat gebruikers logo's die traditioneel linksboven staan ​​89% beter onthouden dan logo's die rechtsboven staan.

[Centered Logos Hurt Website Navigation - NN Group](https://www.nngroup.com/articles/centered-logos/)
In dit artikel wordt beschreven dat gebruikers ongeveer 6 keer lastiger naar de homepage navigeren als het Logo in het midden van de pagina staat, vergeleken met wanneer het in de linkerbovenhoek staat.

[Logo Placement Affects Web Navigation and Brand Recall - NN Group](https://www.nngroup.com/videos/logo-placement/)
In deze video worden bovenstaande artikelen samengevat.

[Homepage Links Remain a Necessity - NN Group](https://www.nngroup.com/articles/homepage-links/)
In dit artikel wordt beschreven dat het nog steeds nodig is om logo's naar de homepage te laten linken (behalve op de homepage zelf). En dat naast het Logo het expliciet aanbieden van een Link 'Home' verstandig is. Deze Link zou onderdeel kunnen zijn van de Main Navigation of de Breadcrumb.

[A link on a logo in the header, what should the alt-text be? - HTMHell](https://htmhell.dev/adventcalendar/2024/1/)
Rian Rietveld beschrijft op welke manier je een Logo in de Page Header moet linken aan de Homepage.

[De toegankelijke naam van een link op een logo in de header](https://nldesignsystem.nl/blog/toegankelijke-naam-link-logo-header/)
Rian Rietveld beschrijft hoe je een link op een logo in de header een goede toegankelijke naam geeft.

## Richtlijnen
[Contrast (Minimum) (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html)
In deze richtlijn [staat aangegeven](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#:~:text=Text%20that%20is%20part%20of%20a%20logo%20or%20brand%20name%20has%20no%20contrast%20requirement.) dat tekst in logo's zich niet aan deze richtlijn hoeft te houden.

[Non-text Contrast (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html)
In deze richtlijn [staat aangegeven](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html#:~:text=The%20graphic%20is%20part%20of%20a%20logo%20or%20brand%20name%20(which%20is%20considered%20%22essential%22%20to%20its%20presentation).) dat het beeldmerk in logo's zich niet aan deze richtlijn hoeft te houden.

**Comment by jeffreylauwers** [comment]
## Amsterdam

- [Storybook](https://designsystem.amsterdam/?path=/docs/components-media-logo--docs)
- [Figma](https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?node-id=6852-7984)
- [GitHub](https://github.com/Amsterdam/design-system/tree/develop/packages/css/src/components/logo)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-10-02 om 15 50 04](https://github.com/user-attachments/assets/9a3044c1-3834-40ba-8756-4425f54568d9)

GGD Amsterdam
![Scherm­afbeelding 2024-10-02 om 15 50 10](https://github.com/user-attachments/assets/81e96bbd-bb65-455e-89d8-0961923ec17b)

Stadsarchief
![Scherm­afbeelding 2024-10-02 om 15 50 13](https://github.com/user-attachments/assets/b071fb04-6569-415e-806b-8259cffde33e)

Stadsbank van Lening
![Scherm­afbeelding 2024-10-02 om 15 50 18](https://github.com/user-attachments/assets/48467c5c-81e0-460e-8697-d205f15a1efd)

VGA Verzekeringen
![Scherm­afbeelding 2024-10-02 om 15 50 22](https://github.com/user-attachments/assets/84a1eea2-012f-4d17-a757-780a6a906490)

Museum Weesp
![Scherm­afbeelding 2024-10-02 om 15 50 26](https://github.com/user-attachments/assets/a82be618-6c84-450d-ba7a-5ec6cdcbd967)

---

### Notities

- Documentatie beschikbaar (Storybook)
- Er worden verschillende verplichtingen genoemd die voor alle logo's gelden:
  - Het logo wordt altijd getoond.
  - Het logo staat links uitgelijnd in de header.
  - Het logo linkt naar de homepage van de website of applicatie.
  - Het logo heeft altijd een hoogte van 40 pixels.


Implementatie (Storybook) is een svg binnen een `<a>` met een visueel verborgen `<span>` met de tekst 'Ga naar de homepage'.

```
<a href="/" class="ams-header__logo-link">
  <span class="ams-visually-hidden">
    Ga naar de homepage
  </span>
  <svg class="ams-logo" aria-hidden="true" focusable="false" viewBox="0 0 115 40" xmlns="http://www.w3.org/2000/svg">
    svg code
  </svg>
</a>
```

**Comment by jeffreylauwers** [comment]
## Den Haag

- [Storybook - binnen Page Header](https://nl-design-system.github.io/denhaag/?path=/story/css-page-elements-header--default)
- [Figma](https://www.figma.com/design/JpoY3waVoQGlLQzQXTL9nn/%F0%9F%8C%BB-Den-Haag-Design-System?node-id=3497-14523&t=Ib4uNxbdToQZIgWJ-4)
- [GitHub - Header logo](https://github.com/nl-design-system/denhaag/tree/main/components/HeaderLogo)

---

### Screenshots / Varianten

**Storybook - binnen Page Header**
![Scherm­afbeelding 2024-10-02 om 16 06 16](https://github.com/user-attachments/assets/1bb77fe1-2a61-41fb-aeab-90fd0f10d5c2)

**Figma**

Den Haag logo variaties
![Scherm­afbeelding 2024-10-02 om 16 07 30](https://github.com/user-attachments/assets/b48e36b5-6131-49b6-8e2e-7183ca19b420)

MijnDenHaag logo variaties
![Scherm­afbeelding 2024-10-02 om 16 07 36](https://github.com/user-attachments/assets/3b948bac-33f5-4d2d-93ab-3947e608d97c)


---

### Notities

Implementatie (Storybook) is een svg binnen een `<a>` met een `arial-label` 'Gemeente Den Haag'. 


```
<div class="denhaag-header__logo">
  <a class="denhaag-logo denhaag-header__link" href="https://www.denhaag.nl" aria-label="Gemeente Den Haag">
    <svg class="denhaag-header__logo" xmlns="http://www.w3.org/2000/svg" width="153" height="48" viewBox="0 0 153 48" fill="none">
    svg code...
    </svg>
  </a>
</div>
```

**Comment by jeffreylauwers** [comment]
## DSO

[Storybook](https://storybook.dso-toolkit.nl/62.8.4/?path=/docs/core-logo--docs)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-10-02 om 16 15 56](https://github.com/user-attachments/assets/490003ca-40c0-422b-817e-e706ce4a2c83)

With Label
![Scherm­afbeelding 2024-10-02 om 16 16 01](https://github.com/user-attachments/assets/7c02cde5-948a-4599-950e-c3582d428b7e)

With (Beta) Ribbon
![Scherm­afbeelding 2024-10-02 om 16 16 06](https://github.com/user-attachments/assets/1707a6a6-a5cb-4f08-b44d-20fcc7e86639)

With Label and Ribbon
![Scherm­afbeelding 2024-10-02 om 16 16 12](https://github.com/user-attachments/assets/154480e1-6ee3-482f-abf6-3809696a920b)

---

### Notities

- Documentatie beschikbaar (Storybook)
- Mogelijkheid om Label en Ribbon toe te voegen.
- Responsive gedrag wordt beschreven.
- Toegankelijkheid: Er wordt aangegeven als er een label en/of een ribbon wordt toegevoegd, deze op het component wordt gezet met een aria-label.

**Comment by jeffreylauwers** [comment]
## DSO

[Documentatie website - binnen Header](https://uno.dfront.rijkscloud.nl/#/components/header)

---

### Screenshots / Varianten

**Documentatie website**

Default
![Scherm­afbeelding 2024-10-02 om 16 26 32](https://github.com/user-attachments/assets/6468e32c-a780-43ad-88bf-229da1b6a1db)

Ministerie van Onderwijs, Cultuur en Wetenschap
![Scherm­afbeelding 2024-10-02 om 16 27 37](https://github.com/user-attachments/assets/7745c53c-f08a-488d-bed0-6936bb613587)

College voor Toetsen en Examens
![Scherm­afbeelding 2024-10-02 om 16 27 44](https://github.com/user-attachments/assets/8e25e8f7-b8d0-4d23-9111-6e0b3f998b87)

Education Executive Agency
(Engelse variant van het DUO logo)
![Scherm­afbeelding 2024-10-02 om 16 27 49](https://github.com/user-attachments/assets/bbda1781-5dd9-4b37-97a6-fb176dfbca55)

---

### Notities

- Als onderdeel van Header component.
- In de documentatie wordt aangegeven dat als de gebruiker op het logo klikt, deze wordt verwezen naar de hoofdpagina van de website.
- Over toegankelijkheid wordt het volgende geschreven:

>Voor de toegankelijkheid van de link met het logo is het niet voldoende om alleen een title op te nemen, omdat screenreaders dat niet altijd goed verwerken. Neem daarom naast de title ook altijd een tekst in de link op.
> 
> Omdat we die tekst niet zichtbaar willen hebben, maar deze er alleen voor screenreaders is, krijgt deze de class screenreader-only . Natuurlijk moet de link tekst een duidelijke beschrijving bevatten.

Implementatie bestaat uit meerdere `<span>` elementen binnen een `<a>` met een `title` 'Logo Dienst Uitvoering Onderwijs'. De eerste 3 `<span>` elementen zorgen voor het beeldmerk, de medium en small tekst. Allen als svg bestand. De 4e `<span>` bevast de visueel verborgen tekst 'Dienst Uitvoering Onderwijs Ministerie van Onderwijs, Cultuur en Wetenschap logo. Naar de homepagina'.

```
<a data-v-7fb6ff2a="" href="#/" class="" title="Logo Dienst Uitvoering Onderwijs">
  <span data-v-7fb6ff2a="" class="header-logo__brand">
    <svg data-v-7fb6ff2a="" class="svg-beeldmerk" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 57.6 100.9">
      scg code...
    </svg>
  </span>
  <span data-v-7fb6ff2a="" class="header-logo__text header-logo__text--medium">
    <svg data-v-7fb6ff2a="" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 221.79 58.6">
      svg code...
    </svg>
  </span>
  <span data-v-7fb6ff2a="" class="header-logo__text header-logo__text--small">
    <svg data-v-7fb6ff2a="" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 114.66 31.23">
      svg code...
    </svg>
  </span>
  <span data-v-7fb6ff2a="" class="screenreader-only">
    Dienst Uitvoering Onderwijs Ministerie van Onderwijs, Cultuur en Wetenschap logo. Naar de homepagina
  </span>
</a>
```

**Comment by jeffreylauwers** [comment]
## OpenGemeenten

OpenGemeenten heeft als leverancier te maken met verschillende logo's. We zijn nu uitgegegaan van het logo dat gebruikt worden binnen de voorbeeld website.

- [Voorbeeld website lenteveld.nl](https://www.lenteveld.nl/)
- [Patternlab](https://patternlab.lenteveld.nl/patterns/01-molecules-001-Generic-Page-Components-20-Page-Header-00-OpenGemeentenSite-PageHeader-Logo-OpenGemeentenSite-PageHeader-Logo/01-molecules-001-Generic-Page-Components-20-Page-Header-00-OpenGemeentenSite-PageHeader-Logo-OpenGemeentenSite-PageHeader-Logo.html)

---

### Screenshots / Varianten

**Voorbeeld website - binnen Page Header**
![Scherm­afbeelding 2024-10-02 om 16 54 38](https://github.com/user-attachments/assets/b1435ddf-4854-4710-b866-c730d8df89dd)

**Patternlab**
![Scherm­afbeelding 2024-10-02 om 16 54 44](https://github.com/user-attachments/assets/88016983-a166-430e-82e0-003a63f9425b)

---

### Notities

Implementatie bestaat uit een `<a>` met daarbinnen een `<picture>` element. Binnen het `<picture>` zijn er 2 `<source>` elementen en 1 `<img>` element. Allen verwijzen naar een svg bestand. Het `<img>` heeft een alt tag met de tekst 'Logo Home'.

```
<a class="page-header__logo" title="Naar homepage" href="1">
  <picture>
    <source srcset="pad naar svg..." media="(max-width: 29.999rem)">
    <source srcset="pad naar svg..." media="(min-width: 30rem)">
    <img src="pad naar svg..." alt="Logo Home">
  </picture>
</a>
```

**Comment by jeffreylauwers** [comment]
## Rotterdam

- [Storybook](https://nl-design-system.github.io/rotterdam/?path=/docs/web-component-rods-logo-image--docs)
- [Figma](https://www.figma.com/design/ZWSC4gCrOXRUR9UX3aoZ8x/Gemeente-Rotterdam-Design-System?node-id=448-13418&t=FvYu1mkAmFztYNKj-4)
- [GitHub](https://github.com/nl-design-system/rotterdam/blob/main/packages/web-components-stencil/src/logo-image.scss)

---

### Screenshots / Varianten

**Storybook**

Base logo
The base logo is formed by the mark and the word mark, executed in landscape mode. If there’s enough space use the base logotype.
![Scherm­afbeelding 2024-10-02 om 17 03 27](https://github.com/user-attachments/assets/64802235-e7ef-44da-ba0b-756a43657ca7)

Square logo
If there isn’t enough space to use the base logo you’re allowed to use the ‘Square logo’.
![Scherm­afbeelding 2024-10-02 om 17 03 56](https://github.com/user-attachments/assets/9dcbbc07-533f-40da-83b7-bcae58c1044d)

Mark
For smaller displays you could use only the mark of Gemeente Rotterdam. Be sure to use the base logo in the footer on the web page or somewhere within the app (splash screen for example).
![Scherm­afbeelding 2024-10-02 om 17 04 02](https://github.com/user-attachments/assets/921d95a3-5f51-4481-b6d3-792b57ab4ac0)

Avatars & favicons
For use in avatars & favicons the element that represents ‘water’ in the mark is removed. This leaves the distinct ‘R’.
![Scherm­afbeelding 2024-10-02 om 17 04 07](https://github.com/user-attachments/assets/926d1b45-b4e5-4f11-9f6b-473f298e0e8e)

---

### Notities

- Verschillende varianten beschikbaar.
- Implementatie zoals toegepast voor '[Mijn loket template](https://nl-design-system.github.io/rotterdam/?path=/docs/template-mijn-loket--docs)'. `<a>` met `aria-label` 'homepage'. Binnen de `<a>` zit een webcomponent met `aria-label` 'logo gemeente Rotterdam'.

```
<a href="#" class="utrecht-link utrecht-link--html-a utrecht-link--box-content" aria-label="homepage">
   <rods-logo-image aria-label="logo gemeente Rotterdam" role="img" class="example-page-header__logo hydrated">
   </rods-logo-image>
</a>
```

**Comment by jeffreylauwers** [comment]
## RVO

- [Documentatie website](https://nl-design-system.github.io/rvo/docs/components/logo/)
- [Storybook](https://nl-design-system.github.io/rvo/?path=/docs/componenten-logo--documentatie)
- [Figma](https://www.figma.com/design/Sj6myBL1Fvot5M1qGxzvEo/ROOS-(RVO-Design-System)?node-id=107-5162)
- [GitHub](https://github.com/nl-design-system/rvo/tree/master/components/logo)

---

### Screenshots / Varianten

**Storybook**

Default
![Scherm­afbeelding 2024-10-02 om 17 15 11](https://github.com/user-attachments/assets/3f20eeaa-a969-4313-8362-3315e5dcfd14)

Alternatieve tekst
![Scherm­afbeelding 2024-10-02 om 17 15 35](https://github.com/user-attachments/assets/dd6e946e-40f6-48b6-a906-0b30b5c740d7)


---

### Notities

In Storybook de mogelijkheid om de 'title' en 'subtitle' zelf in te vullen 🤩

![Scherm­afbeelding 2024-10-02 om 17 26 48](https://github.com/user-attachments/assets/4302a2f2-0cf7-4adc-a77c-8195a5c9305b)



Implementatie zoals toegepast op [Content template](https://nl-design-system.github.io/rvo/?path=/story/pagina-s-templates-content--default). Een `<a>` met daarbinnen een `<div>`. Binnen deze `<div>` zitten 2 `<div>`'s. De eerste `<div>` zorgt voor het beeldmerk (svg). De 2e `<div>` groepeert de 'title' en optionele 'subtitle'. Zowel 'title' en 'subtitle' zijn een `<p>` element.

```
<a href="#" class="rvo-header__logo-link rvo-link rvo-link--no-underline">
  <div class="rvo-logo rvo-header__logo-img">
    <div class="rvo-logo__emblem">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 -12 44 88">
        svg code...
      </svg>
    </div>
    <div class="rvo-logo__wordmark">
      <p class="rvo-logo__title">
        Rijksdienst voor Ondernemend Nederland
      </p>
    </div>
  </div>
</a>
```

**Comment by jeffreylauwers** [comment]
## Logo

- [Storybook - Logo](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/utrecht-logo--docs)
- [Storybook - CSS implementatie](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-logo--docs)
- [Figma](https://www.figma.com/file/msb3CfQBefPoruqNQ968Zh/Utrecht-Design-System?node-id=69%3A907)
- [GitHub](https://github.com/nl-design-system/utrecht/tree/main/components/logo)

---

### Screenshots / Varianten

**Storybook - Logo**

Normaal
Het normale logo in het rood met geel. Gebruik voor desktop webapplicaties en e-mail handtekeningen met donkere achtergrond. Dit logo is het meest herkenbaar en straalt vertrouwen uit.
![Scherm­afbeelding 2024-10-03 om 11 36 37](https://github.com/user-attachments/assets/22eeee88-901b-4c3f-9302-aac0fd95b142)

Normaal zonder balk
Het normale logo in het rood zonder balk. Gebruik voor desktop webapplicaties waar de achtergrond in conflict is met de gele kleur van de balk.
![Scherm­afbeelding 2024-10-03 om 11 36 42](https://github.com/user-attachments/assets/613e88bf-fb53-40c6-9b61-bd347087ddde)

Medium
Kleiner formaat van het logo voor gebruik van apps of andere mobiele applicaties.
![Scherm­afbeelding 2024-10-03 om 11 36 47](https://github.com/user-attachments/assets/a70a1f24-44f6-408c-827e-dd2b9e7daeb9)

Klein
Basic klein formaat van het logo van de Gemeente Utrecht, voor gebruik van avatar of mobiele oplossingen waar ruimte gebrek kan onstaan.
![Scherm­afbeelding 2024-10-03 om 11 36 52](https://github.com/user-attachments/assets/b6cf9d9b-683c-49a0-b544-f77ac2bfdfde)

Favicon
Gebruiken als favicon.
![Scherm­afbeelding 2024-10-03 om 11 36 56](https://github.com/user-attachments/assets/450939d5-9d71-46ad-aff7-37495b644a9f)


---

### Notities

Uitgebreide documentatie beschikbaar (Storybook - CSS implementatie)

Implementatie zoals beschreven in de documentatie wanneer het logo linkt naar de homepage. Een `<a>` met als `aria-label` 'Homepage'. Binnen de `<a>` een `<img>`. Deze `<img>` verwijst naar een `svg` bestand en heeft de alt tekst 'logo Gemeente Utrecht'.

```
<a href="/" aria-label="Homepage" rel="home">
  <img src="logo.svg" alt="logo Gemeente Utrecht" height="100" width="150" />
</a>
```

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Logo op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=logo)
